src/controllers/group.js

```diff
@@ -90,8 +103,12 @@ const Group = {
         `/groups/${id}`,
       ));
     }
-
-    return res.status(200).send(result.toJSON());
+    // eslint-disable-next-line max-len
+    for (let i = 0; i < result.images.length; i += 1) {
+      result.images[i].URL = await S3.getPreSignedURL(`groups/${result._id}/${result.images[i].fileName}`);
+    }
+    result = { result, thumbnail: result.images[0] };
```

Comment:
```suggestion
    result.thumbnail = result.images[0];
```

Reply:
That is what I initially wrote but it doesn't work. The field isn't added.

Reply:
try this instead

```js
const json = result.toJSON();
json.thumbnail = result.images[0];
```